src/hooks/api/places.ts

```diff
@@ -295,8 +303,10 @@ const useGetPlacesByQuery = (
       zip,
       addressCountry,
       addressLocality,
+      streetAddress,
+      searchTerm,
     },
-    enabled: !!name || terms.length > 0,
+    enabled: !!searchTerm || !!streetAddress || terms.length > 0,
```

Comment:
Shouldn't it also be enabled when the `name` is defined?
```suggestion
    enabled: !!name || !!searchTerm || !!streetAddress || terms.length > 0,
```